docs/internals/optimizer.rst

```diff
@@ -26,6 +26,10 @@ One can use ``solc --ir-optimized --optimize`` to produce an
 optimized Yul IR for a Solidity source. Similarly, one can use ``solc --strict-assembly --optimize``
 for a stand-alone Yul mode.
 
+.. note::
+    The `PeepholeOptimizer <https://en.wikipedia.org/wiki/Peephole_optimization>`_ and the inliner are always
+    enabled by default and can only be turned off via the :ref:`JSON-input-output interface <compiler-api>`.
```

Comment:
We usually refer to it as Standard JSON:
```suggestion
    enabled by default and can only be turned off via the :ref:`Standard JSON <compiler-api>`.
```

Reply:
Ok. Done.